src/main/java/arb/logic/commands/project/ListProjectCommand.java

```diff
@@ -32,7 +33,9 @@ public CommandResult execute(Model model, ListType currentListBeingShown) {
         requireNonNull(model);
         model.updateFilteredProjectList(PREDICATE_SHOW_ALL_PROJECTS);
         model.updateSortedProjectList(PROJECT_NO_COMPARATOR);
-        return new CommandResult(MESSAGE_SUCCESS, ListType.PROJECT);
+        String copy = model.getProjectsContent();
+        String message = String.format(MESSAGE_PROJECTS_CONTENT, copy);
```

Comment:
This is short enough that you can flatten it to `String.format(MESSAGE_PROJECTS_CONTENT, model.`);

If you'd like to change it to format three integers, it would just be `String.format(%s %s %s, int, int, int);`

Reply:
ok changed